packages/cli/src/ceramic-daemon.ts

```diff
@@ -143,22 +143,21 @@ class CeramicDaemon {
   }
 
   registerAPIPaths (app: core.Express, gateway: boolean): void {
-    app.get(toApiPath('/records/ceramic/:docid'), this.records.bind(this))
-    app.get(toApiPath('/show/ceramic/:docid'), this.show.bind(this))
-    app.get(toApiPath('/state/ceramic/:docid'), this.state.bind(this))
-    app.get(toApiPath('/pin/ls/ceramic/:docid'), this.listPinned.bind(this))
-    app.get(toApiPath('/pin/ls'), this.listPinned.bind(this))
-    app.get(toApiPath('/supported_chains'), this.getSupportedChains.bind(this))
-    app.post(toApiPath('/create'), this.createDocFromGenesis.bind(this))
+    app.get(toApiPath('/records/:docid'), this.records.bind(this))
+    app.put(toApiPath('/document'), this.createDocFromGenesis.bind(this))
```

Comment:
Dropping the 'ceramic' from all these API urls makes sense to me, but I'm not sure why we'd want to consolidate all the APIs in a way that removes semantic meaning from the URLs?  Why is `/document` better than `/document/create`?  The latter makes it more immediately obvious what is happening from just the URL being called.  Also having the same URL endpoints do different things based on the HTTP method called seems like a potential source of confusion with no real benefit?  Why not separate `/document/create` and `/document/apply_record`?

Reply:
@simonovic86 recommended that this was the way to go.